go/felix/proto/felixbackend.proto

```diff
@@ -145,8 +145,9 @@ message PolicyID {
 }
 
 message Policy {
-  repeated Rule inbound_rules = 1;
-  repeated Rule outbound_rules = 2;
+  bool untracked = 1;
```

Comment:
As a general rule, (for back-compat) you shouldn't change the field indexes of protobuf fields.  Even if this new field is written first for clarity, you should give it index 3.